README.md

```diff
@@ -282,6 +282,54 @@ $auth->verify($idToken, [
 [google-id-tokens]: https://developers.google.com/identity/sign-in/web/backend-auth
 [iap-id-tokens]: https://cloud.google.com/iap/docs/signed-headers-howto
 
+## Caching
+We currently have limited support for caching. Caching is enabled by passing a PSR-6 `CacheItemPoolInterface` instance to the constructor when instantiating the credentials.
```

Comment:
nit: line lengths. With the exception of URLs, we should not exceed 80 characters per line